config/rbac/role.yaml

```diff
@@ -6,29 +6,6 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
-- apiGroups:
```

Comment:
@mkumatag so we do not need those rules anymore for secrets and events?

Reply:
na! I think this is happening part of autogenerated code, lemme fix the annotations in the controller to keep these rules intact.

Reply:
@gyliu513 added the kubebuilder rule to the controller with the required permissions. ptal at latest commit